src/utils/constants.ts

```diff
@@ -1,68 +1,38 @@
-export const DECK_HANDS = [
-  '2',
-  '3',
-  '4',
-  '5',
-  '6',
-  '7',
-  '8',
-  '9',
-  '10',
-  'J',
-  'Q',
-  'K',
-  'A',
-];
-export const DEFAULT_TURN_DURATION = 300;
-export const DECK_SUITS = ['A', 'B', 'C', 'D', 'E', 'T'] as const;
-export type DeckSuit = typeof DECK_SUITS[number];
-export const TRUMP_CARD_PREFIX = 'T';
-export const DEFAULT_START_SUIT = 'E';
-export const GAMEMODE_EOP = 'Elevation of Privilege';
-export const GAMEMODE_CORNUCOPIA = 'OWASP Cornucopia';
-export const DEFAULT_GAME_MODE = GAMEMODE_EOP;
-export type GameMode = typeof GAMEMODE_EOP | typeof GAMEMODE_CORNUCOPIA;
-export const INVALID_CARDS = {
-  [GAMEMODE_EOP]: ['E2', 'T2', 'T3', 'T4'],
-  [GAMEMODE_CORNUCOPIA]: [],
-};
-export const STARTING_CARD_MAP = {
-  [GAMEMODE_EOP]: {
-    D: 'D2',
-    E: 'E3',
-    C: 'C2',
-    B: 'B2',
-    A: 'A2',
-    T: 'T5',
-  },
-  [GAMEMODE_CORNUCOPIA]: {
-    D: 'D2',
-    E: 'E2',
-    C: 'C2',
-    B: 'B2',
-    A: 'A2',
-    T: 'T2',
-  },
-};
+export enum GameMode {
+  EOP = 'Elevation of Privilege',
+  CORNUCOPIA = 'OWASP Cornucopia',
+}
+
+export const DEFAULT_GAME_MODE = GameMode.EOP;
 
+export function isGameMode(value: GameMode): value is GameMode {
+  return value in GameMode;
+}
+
+export function isGameModeCornucopia(gameMode: GameMode): boolean {
+  return isGameMode(gameMode) && gameMode === GameMode.CORNUCOPIA;
+}
+
+export enum ModelType {
+  THREAT_DRAGON = 'Threat Dragon',
+  DEFAULT = 'Default',
+  IMAGE = 'Image',
+}
+
+export const DEFAULT_START_SUIT = 'E';
 export const CARD_LIMIT = 26;
+
 export const MIN_NUMBER_PLAYERS = 2;
 export const MAX_NUMBER_PLAYERS = 9;
+export const DEFAULT_TURN_DURATION = 300;
 
+//TODO: Move this into dedicated server config file
```

Comment:
we should either address this here or create an issue for it.

Reply:
Moved this into a dedicated config file.